components/eam/bld/build-namelist

```diff
@@ -550,9 +550,17 @@ if ($dyn =~ /se/) {add_default($nl, 'vect_map', 'val'=>'cart3d');}
 
 # Adiabatic or Ideal physics
 my $phys = $cfg->get('phys');
+# TODO: I do not think these are used in E3SM anymore, and should be removed.
```

Comment:
did you mean to make this comment just about the adiabatic mode?

Reply:
Not exactly, I think this was before I discovered that these were used in the mct driver, not in EAM. Removing comment.